docs/quickstart.md

```diff
@@ -0,0 +1,766 @@
+---
+title: QuickStart
+description: Powertools introduction
+---
+Quickstart introducing core Powertools functionalities.
+
+
+## Installation 
```

Comment:
As you're covering **1/** Cloning the project, **2/** Making small modifications, **3/** Running it locally, you're past the scope of an Installation header -- **Getting started** makes this more explicit and gives you room for additional changes you might want to make.

```suggestion
## Getting started 
```

Reply:
Changed as suggested